openapi3/refs_test.go

```diff
@@ -332,3 +334,65 @@ components:
 	_, _, err = ptr.Get(doc)
 	require.Error(t, err)
 }
+
+func TestSchemaRef_Extensions(t *testing.T) {
```

Comment:
Yup please integrate with `ref.sh` and also have this test be generated per type as much as you can